docs/siem/detections/building-block-rule.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[building-block-rule]]
+[role="xpack"]
+== About building-block rules
+
+Create building-block rules when you do not want to see their generated alerts
+in the UI. This is useful when you want:
+
+* A record of low-risk alerts without producing noise in the Alerts table.
+* Rules that execute on the alert indices (`.siem-signals-<kibana space>-*`).
+You can then use building-block rules to create hidden alerts that act as a
+basis for an 'ordinary' rule to generate visible alerts.
+
+[float]
+=== Set up rules that run on alert indices
+
+To create a rule that searches alert indices, in the *Index patterns* field,
+add the index pattern for alert indices:
+
+[role="screenshot"]
+image::images/alert-indices-ui.png[]
+
+[float]
+
+=== View building-block alerts in the UI
+
+. Go to *Security* -> *Detections*
+. In the Alert table, select _Additional filters_ ->
+_Include building-block alerts_.
+
+NOTE: On a building-block Rule details page, the rule's alerts are displayed (by
```

Comment:
does "Details" need to be capitalized as well if we're referring to the "Rule Details" page?

Reply:
No - sentence capitalisation